types/hotwired__turbo/package.json

```diff
@@ -16,6 +16,10 @@
         {
             "name": "Luke Davia",
             "githubUsername": "lukeify"
+        },
+        {
+            "name": "Alex Coles",
```

Comment:
Happy to have you on board as an owner, but I'd prefer that be done in a dedicated PR

Reply:
Sounds entirely reasonable! - will drop this commit.